nservicebus/rabbitmq/rabbitmqclient-nuget.md

```diff
@@ -0,0 +1,32 @@
+---
+title: RabbitMQ.Client Nuget usage
+summary: Explains why the RabbitMQ.Client Nuget version range is inconsistent with most other NServiceBus extension nugets.
+---
+
+The NServiceBus.RabbitMQ library uses the [RabbitMQ.Client](http://www.nuget.org/packages/RabbitMQ.Client/) to communicate with the underlying queueing system. Due to the API/nuget versioning strategy of RabbitMQ.Client there are some complexities to targeting that library from the perspective of in intermediary library like NServiceBus.RabbitMQ.

```

Comment:
I'd keep it simple and change the 2nd sentence to say: `However, due to the API/nuget versioning strategy of RabbitMQ.Client there are some complexities involved when referencing this dll from external libraries like NServiceBus.RabbitMQ`